workflows-scripts/circulation/check-in-check-out/circulation_checkInCheckOut.jmx

```diff
@@ -15,7 +15,7 @@
         <collectionProp name="Arguments.arguments">
           <elementProp name="HOSTNAME" elementType="Argument">
             <stringProp name="Argument.name">HOSTNAME</stringProp>
-            <stringProp name="Argument.value">${__P(HOSTNAME,)}</stringProp>
+            <stringProp name="Argument.value">${__P(HOSTNAME,[host-name])}</stringProp>
```

Comment:
`HOSTNAME` is already stored above. Do we still need `[hostname]`?

Reply:
It's just a placeholder indicating the real value. We once had something similar to it but I removed it by accident.